clients/js/README.md

```diff
@@ -1,97 +1,95 @@
 # Wormhole CLI
 
-
+This tool is a command line interface to Wormhole.
 ## Installation
 
     make install
 
 This installs two binaries, `worm-fetch-governance` and `worm` on your `$PATH`.
 
-## Usage
-
-### `worm-fetch-governance`
-
-    Usage:
-      worm-fetch-governance [sequence]
-
-    Fetch a governance VAA by sequence number, and print it as hex.
-
-
-For example
-
-    worm-fetch-governance 13940208096455381020
-
-prints
-
-    01000000010d0012e6b39c6da90c5dfd3c228edbb78c7...
-
-
-### `worm`
-
-This is the main CLI tool. To use it, set up `$HOME/.wormhole/.env` with your
+To use `worm`, set up `$HOME/.wormhole/.env` with your
 private keys, based on `.env.sample` in this folder.
 
-    worm [command]
+## Usage
+
+``` sh
+worm [command]
 
-    Commands:
-      worm generate      generate VAAs (devnet and testnet only)
-      worm parse <vaa>   Parse a VAA
-      worm submit <vaa>  Execute a VAA
+Commands:
+  worm generate                             generate VAAs (devnet and testnet
+                                            only)
+  worm parse <vaa>                          Parse a VAA
+  worm recover <digest> <signature>         Recover an address from a signature
+  worm contract <network> <chain> <module>  Print contract address
+  worm rpc <network> <chain>                Print RPC address
+  worm evm                                  EVM utilites
+  worm submit <vaa>                         Execute a VAA
 
-    Options:
-      --help     Show help                                  [boolean]
-      --version  Show version number                        [boolean]
+Options:
+  --help     Show help                                                 [boolean]
+  --version  Show version number                                       [boolean]
+```
```

Comment:
--network should be added here 
maybe allow for passing in *any* gaurdian endpoint so people can use this against their own networks?

Reply:
Agreed that `--network` should be a global flag (right now most commands take it, but not all - so didn't want to make it a global required flag. That said, we can just declare the flag globally as optional, then do the requirement check only in the commands that actually need it). Happy to come back to this in another PR (see my other comment below). FWIW the CLI currently offers no commands that communicate with guardian endpoints, the network flag instead is used for selecting contract addresses, and chain rpc endpoints. So when you say `worm submit <vaa>`, the chain id in the VAA determines which chain to submit to, and the `--network` flag is then used to look up what the contract address and chain RPC endpoint are. Overriding the contract address and the RPC is possible in certain subcommands, though we could probably take a more methodical approach to these flags.

Something that would be interesting is overriding these values from a json config file rather than through flags.